docs/operations/rule-configuration.md

```diff
@@ -153,9 +153,9 @@ Set the following properties:
 
 - `period`: a JSON object representing [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) periods. The period is from some time in the past to the present, or into the future if `includeFuture` is set to `true`.
 - `includeFuture`: a boolean flag to instruct Druid to match a segment if:
-<br>- the segment interval overlaps the rule interval, or
-<br>- the segment interval starts any time after the rule interval starts.
-<br>You can use this property to load segments with future start and end dates, where "future" is relative to the time when the Coordinator evaluates data against the rule. Defaults to `true`.
+<br />- the segment interval overlaps the rule interval, or
+<br />- the segment interval starts any time after the rule interval starts.
+<br />You can use this property to load segments with future start and end dates, where "future" is relative to the time when the Coordinator evaluates data against the rule. Defaults to `true`.
```

Comment:
At the cost of extra per-line spacing, you can replace the `<br />` above with two spaces and get the same effect in a more Markdown-ish way. Here and below.